pages/world/[id].tsx

```diff
@@ -115,7 +114,7 @@ function WorldPage() {
 
     return levels.map((level, index) => new SelectOption(
       level.name,
-      `/level/${level._id.toString()}?wid=${id}`,
+      `/level/${level.slug}?wid=${id}`,
```

Comment:
Need to do this on the universe page as well since it now displays worlds + levels